modules/tests/test/testconfigs/create-data-object-test-config.go

```diff
@@ -127,6 +140,34 @@ func (c *CreateDataObjectTestConfig) GetTaskRun() *v1beta1.TaskRun {
 						StringVal: ToStringBoolean(c.TaskData.AllowReplace),
 					},
 				},
+				{
+					Name: CreateDataObjectParams.DeleteObject,
+					Value: v1beta1.ArrayOrString{
+						Type:      v1beta1.ParamTypeString,
+						StringVal: ToStringBoolean(c.TaskData.DeleteObject),
+					},
+				},
+				{
+					Name: CreateDataObjectParams.DeleteObjectName,
+					Value: v1beta1.ArrayOrString{
+						Type:      v1beta1.ParamTypeString,
+						StringVal: c.TaskData.Name,
+					},
+				},
+				{
+					Name: CreateDataObjectParams.DeleteObjectKind,
+					Value: v1beta1.ArrayOrString{
+						Type:      v1beta1.ParamTypeString,
+						StringVal: c.TaskData.ObjectKind,
+					},
+				},
+				{
+					Name: CreateDataObjectParams.DataObjectNamespace,
+					Value: v1beta1.ArrayOrString{
+						Type:      v1beta1.ParamTypeString,
+						StringVal: c.TaskData.dataObjectNamespace,
```

Comment:
Can't we just reuse `c.TaskData.Namespace` here?

Reply:
To use `c.TaskData.Namespace`, you have to call `options.ResolveNamespace(c.TaskData.Namespace, "")`, but `options` variable is only available in `Init` function. The `c.TaskData.dataObjectNamespace` is only internal variable, user creating tests can't modify it from the test.